webapp/src/main/java/com/box/l10n/mojito/service/delta/DeltaService.java

```diff
@@ -123,16 +122,14 @@ public DeltaResponseDTO getDeltasForRuns(
     List<Long> pushRunIds = getIds(pushRuns);
     List<Long> pullRunIds = getIds(pullRuns);
 
-    DateTime translationsFromDate =
+    ZonedDateTime translationsFromDate =
         Optional.ofNullable(pullRuns).orElse(Collections.emptyList()).stream()
             .min(Comparator.comparing(PullRun::getCreatedDate))
             .map(PullRun::getCreatedDate)
             // Remove milliseconds as the Mojito DB does not store dates with sub-second precision.
-            .map(dateTime -> dateTime.withMillisOfSecond(0))
-            .orElse(new DateTime(0));
-    Instant fromDateInstant = Instant.ofEpochMilli(translationsFromDate.getMillis());
-    Timestamp sqlTranslationsFromDate =
-        Timestamp.valueOf(LocalDateTime.ofInstant(fromDateInstant, ZoneOffset.UTC));
+            .map(dateTime -> dateTime.withNano(0))
+            .orElse(ZonedDateTime.ofInstant(Instant.EPOCH, ZoneId.systemDefault()));
```

Comment:
If UTC was previously specified in the deleted line shouldn't we re-use it again?